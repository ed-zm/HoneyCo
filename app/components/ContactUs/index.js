import React from 'react'
import styled from 'styled-components'
import { Row, Container } from 'reactstrap'
import { HoneyCoLightGray, HoneyCoDarkGray, HoneyCoYellow } from '../../constants'

const StyledDiv = styled.div`
  background-color: #EFEFEF;
  width: 100%;
  padding: 2em;
  padding-bottom: 3em;
  border-bottom: 10px solid ${HoneyCoDarkGray};
`
const StyledTextContainer = styled.div`
  font-size: 20px;
  width: 100%;
`
const StyledFirstText = styled.span`
  display: block;
  width: 100%;
  text-align: center;
  font-weight: bold;
`
const StyledSecondText = styled.span`
  display: block;
  width: 100%;
  text-align: center;
  margin: 1em 0;
`
const StyledInputsContainer = styled.div`
  font-size: 20px;
  margin: 0 auto;
  width: 40%;
`
const StyledInput = styled.input`
  font-family: Oswald;
  width: 100%;
  border-radius: 12px;
  border: none;
  height: 50px;
  color: ${HoneyCoLightGray};
  margin: 1em 0;
`
const StyledEmailInput = styled.input`
  font-family: Oswald;
  width: 75%;
  border-radius: 12px 0px 0px 12px;
  border: none;
  height: 50px;
  color: ${HoneyCoLightGray};
`
const StyledButton = styled.button`
  width: 25%;
  font-size: 20px;
  font-family: Oswald;
  display: inline-block;
  vertical-align: bottom;
  height: 50px;
  border-radius: 0px 12px 12px 0px;
  border: none;
  color: white;
  font-wegiht: 500;
  background-color: ${HoneyCoYellow};
`
const ContactUs = () =>
  <StyledDiv>
    <Container>
      <Row>
        <StyledTextContainer>
          <StyledFirstText>Send Us a Message</StyledFirstText>
          <StyledSecondText>Whether you're interested in living in a HoneyCo Home, or just want periodic updates, we'd love to hear from you.</StyledSecondText>
        </StyledTextContainer>
        <StyledInputsContainer>
          <StyledInput placeholder = '  How can we help you?'/>
          <StyledEmailInput placeholder = ' Enter your email'/>
          <StyledButton>SUBMIT</StyledButton>
        </StyledInputsContainer>
      </Row>
    </Container>
  </StyledDiv>

export default ContactUs
