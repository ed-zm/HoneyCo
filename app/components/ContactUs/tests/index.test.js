import React from 'react'
import { shallow } from 'enzyme'
import ContactUs from '../index'

describe('<ContactUs />', () => {
  it('It should render the ContactUs Component', () => {
    const wrapper = shallow(<ContactUs/>)
    expect(wrapper).toMatchSnapshot()
  })
})
