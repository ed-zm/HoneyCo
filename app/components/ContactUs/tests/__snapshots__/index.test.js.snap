// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactUs /> It should render the ContactUs Component 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <styled.div>
    <gt
        tag="div"
    >
        <bt
            tag="div"
        >
            <styled.div>
                <styled.span>
                    Send Us a Message
                </styled.span>
                <styled.span>
                    Whether you're interested in living in a HoneyCo Home, or just want periodic updates, we'd love to hear from you.
                </styled.span>
            </styled.div>
            <styled.div>
                <styled.input
                    placeholder="  How can we help you?"
                />
                <styled.input
                    placeholder=" Enter your email"
                />
                <styled.button>
                    SUBMIT
                </styled.button>
            </styled.div>
        </bt>
    </gt>
</styled.div>,
  "nodes": Array [
    <styled.div>
      <gt
            tag="div"
      >
            <bt
                  tag="div"
            >
                  <styled.div>
                        <styled.span>
                              Send Us a Message
                        </styled.span>
                        <styled.span>
                              Whether you're interested in living in a HoneyCo Home, or just want periodic updates, we'd love to hear from you.
                        </styled.span>
                  </styled.div>
                  <styled.div>
                        <styled.input
                              placeholder="  How can we help you?"
                        />
                        <styled.input
                              placeholder=" Enter your email"
                        />
                        <styled.button>
                              SUBMIT
                        </styled.button>
                  </styled.div>
            </bt>
      </gt>
</styled.div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <ContactUs />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <styled.div>
          <gt
                    tag="div"
          >
                    <bt
                              tag="div"
                    >
                              <styled.div>
                                        <styled.span>
                                                  Send Us a Message
                                        </styled.span>
                                        <styled.span>
                                                  Whether you're interested in living in a HoneyCo Home, or just want periodic updates, we'd love to hear from you.
                                        </styled.span>
                              </styled.div>
                              <styled.div>
                                        <styled.input
                                                  placeholder="  How can we help you?"
                                        />
                                        <styled.input
                                                  placeholder=" Enter your email"
                                        />
                                        <styled.button>
                                                  SUBMIT
                                        </styled.button>
                              </styled.div>
                    </bt>
          </gt>
</styled.div>,
        "_debugID": 2,
        "_renderedOutput": <styled.div>
          <gt
                    tag="div"
          >
                    <bt
                              tag="div"
                    >
                              <styled.div>
                                        <styled.span>
                                                  Send Us a Message
                                        </styled.span>
                                        <styled.span>
                                                  Whether you're interested in living in a HoneyCo Home, or just want periodic updates, we'd love to hear from you.
                                        </styled.span>
                              </styled.div>
                              <styled.div>
                                        <styled.input
                                                  placeholder="  How can we help you?"
                                        />
                                        <styled.input
                                                  placeholder=" Enter your email"
                                        />
                                        <styled.button>
                                                  SUBMIT
                                        </styled.button>
                              </styled.div>
                    </bt>
          </gt>
</styled.div>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ContactUs />,
}
`;
